app/layout/scenarios/edit/analysis/blm-calibration/component.tsx

```diff
@@ -0,0 +1,219 @@
+            <form
+              className="flex flex-col flex-grow w-full mt-5 overflow-hidden text-gray-500"
+              autoComplete="off"
+              noValidate
+              onSubmit={handleSubmit}
+            >
+              <div className="flex space-x-7">
+                <div className="flex items-center">
+                  <Label theme="dark" className="mr-3 text-xs uppercase">From</Label>
+                  <div className="w-22">
+                    <FieldRFF
+                      name="blmCalibrationFrom"
+                      validate={composeValidators([{ presence: true }])}
+                    >
+                      {(fprops) => (
+                        <Field id="blmCalibrationFrom" {...fprops}>
+                          <Input
+                            mode="dashed"
+                            className="text-2xl"
+                            type="number"
+                            min={minBlmValue}
+                            max={maxBlmValue}
+                            onChange={(e) => {
+                              if (!e.target.value) {
+                                return fprops.input.onChange(null);
+                              }
+                              return fprops.input.onChange(+e.target.value);
+                            }}
+                          />
+                        </Field>
+                      )}
+                    </FieldRFF>
+                  </div>
+                  <p className="ml-5 text-sm text-white opacity-60">{`min ${format(',d')(minBlmValue)}`}</p>
+                </div>
+
+                <div className="flex items-center">
+                  <Label theme="dark" className="mr-3 text-xs uppercase">To</Label>
+                  <div className="w-22">
+                    <FieldRFF
+                      name="blmCalibrationTo"
+                      validate={composeValidators([{ presence: true }])}
+                    >
+                      {(fprops) => (
+                        <Field id="blmCalibrationTo" {...fprops}>
+                          <Input
+                            mode="dashed"
+                            className="text-2xl"
+                            type="number"
+                            min={minBlmValue}
+                            max={maxBlmValue}
+                            onChange={(e) => {
+                              if (!e.target.value) {
+                                return fprops.input.onChange(null);
+                              }
+                              return fprops.input.onChange(+e.target.value);
+                            }}
+                          />
+                        </Field>
+                      )}
+                    </FieldRFF>
+                  </div>
+                  <p className="ml-5 text-sm text-white opacity-60">{`max ${format(',d')(maxBlmValue)}`}</p>
+                </div>
+              </div>
+
+              {!blmGraph && (
```

Comment:
Once I click on the button I can't calibrate again because it depends on the blmGraph? What would happen if after getting the calibration and the graph I want to calibrate it again?